### DISCUSSION TITLE
why next.js example deleted with-immutable ?

### DISCUSSION THREAD

**Discussion by zhangwei900808** [vercel/next.js#16953] [Help]
I want to used immutable.js before I can see this example ,but now I cant see it , next.js deleted?

**Comment by TommySorensen** [comment]
Maybe you can take a look in the old commits. I just did quick search to find something with `immutable`.
https://github.com/vercel/next.js/commit/126507cc1c7c9ab87e9cc826a74840bc63af2eca

**Reply by zhangwei900808** [reply]
why they deleted this commits?
